src/pages/ForgotPasswordPage/ForgotPasswordPage.styled.js

```diff
@@ -0,0 +1,63 @@
+import { Box, Typography, styled } from '@mui/material'
+import backgroundImage from 'Images/cannabis.png'
+
+export const Background = styled(Box)(() => ({
```

Comment:
if you don't have any states here, you should use `styled(Box)({ ... })` instead of this one